src/NOTD.SC2Map/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -95,9 +95,44 @@
         <EditorCategories value=""/>
         <Behavior value="Detector5"/>
     </CEffectApplyBehavior>
+    <CEffectCreateUnit id="CreateNapalmFireInfernoChain">
+        <EditorCategories value=""/>
+        <WhichLocation Value="SourceUnit"/>
+        <Origin Value="SourceUnit"/>
+        <SpawnEffect value="InfernoApplyChainNapalm"/>
+        <SpawnUnit value="NapalmFire"/>
+        <TypeFallbackUnit Value="Source"/>
+    </CEffectCreateUnit>
     <CEffectDamage id="HotSBanelingExplodeLargeSearchDamage">
         <Amount value="60"/>
     </CEffectDamage>
+    <CEffectApplyBehavior id="InfernoApplyChainNapalm">
+        <EditorCategories value=""/>
+        <Behavior value="InfernoChainNapalm"/>
+    </CEffectApplyBehavior>
+    <CEffectApplyBehavior id="InfernoApplyChainTarget">
+        <EditorCategories value=""/>
+        <Behavior value="InfernoChainTarget"/>
+    </CEffectApplyBehavior>
+    <CEffectEnumArea id="InfernoSearchTargetChain">
+        <EditorCategories value=""/>
+        <Chance value="0.4"/>

```

Comment:
^ "0.4" = "40% chance per second for inferno napalm to spread inferno to nearby units"